integration-cli/docker_cli_run_test.go

```diff
@@ -404,8 +446,15 @@ func (s *DockerSuite) TestRunVolumesFromSymlinkPath(c *check.C) {
 }
 
 func (s *DockerSuite) TestRunExitCode(c *check.C) {
-	testRequires(c, DaemonIsLinux)
-	_, exit, err := dockerCmdWithError("run", "busybox", "/bin/sh", "-c", "exit 72")
+	var (
+		exit int
+		err  error
+	)
+	if daemonPlatform == "windows" {
+		_, exit, err = dockerCmdWithError("run", WindowsBaseImage, "cmd", "/s", "/c", "exit 72")
```

Comment:
Why? (Sorry, not sure really that's relevant as most people won't in reality be testing busybox on Windows, but *will* be using cmd and more specifically running containers checking exit codes from cmd. IMO this is a more realistic test on Windows).